Doc/whatsnew/3.10.rst

```diff
@@ -435,6 +435,13 @@ py_compile
 Added ``--quiet`` option to command-line interface of :mod:`py_compile`.
 (Contributed by Gregory Schevchenko in :issue:`38731`.)
 
+pyclbr
+------
```

Comment:
```suggestion
------

```

Reply:
Somehow, a blank line is added to the resulting .html without the blank in the .rst.  But all other entries in the source .rst do have the explicit blank line.  This should too.  Thanks for catching.

Reply:
That must mean that docutils is ok to parse the heading here but github isn’t (I suggested this change after seeing bad colorization on the lines that follow)